arrabiata/src/constraints.rs

```diff
@@ -55,6 +55,13 @@ impl<Fp: PrimeField> InterpreterEnv for Env<Fp> {
         pos
     }
 
+    fn access_next_row(&self, pos: Self::Position) -> Self::Variable {
```

Comment:
In a future PR, I will add the "row" (i.e. Curr or Next) in the type `Self::Position`. It will simplify this code.